ui/shared/theme-editor/react/PropTypes.js

```diff
@@ -19,7 +19,7 @@
 import PropTypes from 'prop-types'
 import _ from 'underscore'
 
-const MD5_REGEX = /[0-9a-fA-F]{32}$/
+const MD5_REGEX = /[0-9a-fA-F]{32,255}$/
```

Comment:
why 255? shouldn't it be exactly 32, or exactly 64?

Reply:
This is fixed.  I also added a '^' anchor.